tests/core/string/test_translation_server.h

```diff
@@ -130,6 +162,25 @@ TEST_CASE("[TranslationServer] Comparing locales") {
 	res = ts->compare_locales(locale_a, locale_b);
 
 	CHECK(res == 0);
+
+	locale_a = "zh-HK";
+	locale_b = "zh";
+
+	// In full standardization, zh-HK becomes zh_Hant_HK and zh becomes
+	// zh_Hans_CN. Both script and country mismatch (-2).
+	res = ts->compare_locales(locale_a, locale_b);
+
+	CHECK(res == 3);
+
+	// In full standardization, zh and zh-CN become zh_Hans_CN for a full
+	// match.
+	locale_a = "zh-CN";
+	locale_b = "zh";
+
+	// No match.
```

Comment:
```suggestion
	// Exact match.
```

Reply:
I put the expanded comment in the wrong spot for this one.